pulp_container/app/serializers.py

```diff
@@ -360,12 +382,36 @@ class Meta:
         fields = tuple(set(DistributionSerializer.Meta.fields) - {"base_url"}) + (
             "repository_version",
             "registry_path",
+            "remote",
```

Comment:
why it needs remote?

Reply:
Same reasoning: https://github.com/pulp/pulp_container/pull/1299/files#r1399046429.